chromium_src/chrome/browser/ui/views/side_panel/side_panel_coordinator.h

```diff
@@ -31,13 +31,11 @@
   friend class BraveSidePanelCoordinator; \
   virtual std::unique_ptr<views::View> CreateHeader
 
-#define UpdateToolbarButtonHighlight virtual UpdateToolbarButtonHighlight
```

Comment:
Our override has already been updated not to be `override` and is being called from our code.